src/pages/Profile.jsx

```diff
@@ -17,7 +17,7 @@ import { useTheme } from '@mui/material/styles';
 import { DocumentListContext } from '@contexts';
 // Component Imports
 import { ConfirmationModal, UploadDocumentModal, SetAclPermissionsModal } from '@components/Modals';
-import { DocumentTable } from '@components/Documents';
+import DocumentTable from '@components/Documents';
```

Comment:
We should be able to delete `DocumentTableRow.jsx` with this PR.

Reply:
Yes, I agree. With the new grid, it's no longer needed.